index.js

```diff
@@ -33,16 +32,14 @@ stops
   .forEach(obs =>
     obs.subscribe(([station, { avgDelay, departures }]) => {
       console.log(station.name, departures);
+      playEnv(avgDelay, station.id);
```

Comment:
Maybe we could have a nicer name for this one?
Like `playStationUpdate`?

Reply:
Definitely, will change that. Actually I didnt exactly imagine to merge this PR so I didnt put so much thought into things, but if you guys like it we should definitely take a closer look at the code and so on.

Reply:
Oh yeah sure.
Honestly I think the repo is in a very WIP state no matter what, so I don't think it's a big deal to merge in things that aren't super thought-through.
As it's a pretty experimental project I think it makes sense it evolves weirdly :P 
But yeah we can discuss it as a group on the next meeting 👍